launcher/src/main/scripts/bin/launcher.py

```diff
@@ -440,6 +441,7 @@ def main():
     o.launcher_log = realpath(options.launcher_log_file or pathjoin(o.data_dir, 'var/log/launcher.log'))
     o.server_log = realpath(options.server_log_file or pathjoin(o.data_dir, 'var/log/server.log'))
 
+    o.jvm_options = options.jvm_options or ()
```

Comment:
Let's move this above around line 428, since this code block is currently just for properties, and this is unrelated.

Reply:
This needs to be `[]` or it fails with
```
can only concatenate list (not "tuple") to list